omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -41,6 +41,9 @@
 
 default['private_chef']['fips_enabled'] = ChefConfig.fips?
 
+default['private_chef']['required_recipe']['enable'] = false
```

Comment:
I dunno, the file existing and the feature being enabled/disabled are two different things. But yeah, YAGNI. The idea here is to have an implementation agnostic switch and the `path` is definitely not agnostic.